src/Templating/Helper/BlockHelper.php

```diff
@@ -325,13 +322,14 @@ private function getEventBlocks(BlockEvent $event): array
         return $results;
     }
 
-    /**
-     * @internal since sonata-project/block-bundle 3.16
-     */
     private function getEventListeners(string $eventName): array
     {
         $results = [];
 
+        if (!$this->eventDispatcher instanceof \Symfony\Component\EventDispatcher\EventDispatcherInterface) {
```

Comment:
Could we import this class?

Reply:
We could use an alias, because we have an existing interface with the same name